### ISSUE TITLE
Trigger loadOptions on command

### ISSUE THREAD

**Issue by keironstoddart** [rob-balfre/svelte-select#521]
Apologies if this is a poor question, I am new to Svelte - but is there anyone way to trigger loadOptions on an event outside of the specific Select component?

**Comment by rob-balfre** [comment]
No, you'd need to manually fetch and populate items. Create a REPL of what you're trying to achieve and I can help more. Thanks
